components/renku_data_services/session/constants.py

```diff
@@ -23,3 +23,6 @@
 
 BUILD_RUN_DEFAULT_RETENTION_AFTER_SUCCEEDED: Final[timedelta] = timedelta(minutes=5)
 """The default retention TTL for BuildRuns when in succeeded state."""
+
+BUILD_RUN_BUILD_TIMEOUT: Final[timedelta] = timedelta(hours=1)
```

Comment:
nitpick about naming:
```suggestion
BUILD_RUN_DEFAULT_TIMEOUT: Final[timedelta] = timedelta(hours=1)
```